packages/components/src/composite/stories/index.story.tsx

```diff
@@ -91,116 +88,112 @@ const meta: Meta< typeof UseCompositeStorePlaceholder > = {
+					<Composite.Item role="gridcell">Item B3</Composite.Item>
+				</Composite.Row>
+				<Composite.Row role="row">
+					<Composite.Item role="gridcell">Item C1</Composite.Item>
+					<Composite.Item role="gridcell">Item C2</Composite.Item>
+					<Composite.Item role="gridcell">Item C3</Composite.Item>
+				</Composite.Row>
+			</>
+		),
+	},
 };
 
-export const Hover: StoryFn< typeof UseCompositeStorePlaceholder > = (
-	storeProps
-) => {
-	const rtl = isRTL();
-	const store = useCompositeStore( { rtl, ...storeProps } );
-
-	return (
-		<Composite store={ store }>
-			<Composite.Hover render={ <Composite.Item /> }>
-				Hover item one
-			</Composite.Hover>
-			<Composite.Hover render={ <Composite.Item /> }>
-				Hover item two
-			</Composite.Hover>
-			<Composite.Hover render={ <Composite.Item /> }>
-				Hover item three
-			</Composite.Hover>
-		</Composite>
-	);
-};
-Hover.parameters = {
-	docs: {
-		description: {
-			story: 'Elements in the composite widget will receive focus on mouse move and lose focus to the composite base element on mouse leave.',
+export const Hover: StoryObj< typeof Composite > = {
+	...Default,
+	args: {
+		children: (
+			<>
+				<Composite.Hover render={ <Composite.Item /> }>
+					Hover item one
+				</Composite.Hover>
+				<Composite.Hover render={ <Composite.Item /> }>
+					Hover item two
+				</Composite.Hover>
+				<Composite.Hover render={ <Composite.Item /> }>
+					Hover item three
+				</Composite.Hover>
+			</>
+		),
+	},
+	parameters: {
+		docs: {
+			description: {
+				story: 'Elements in the composite widget will receive focus on mouse move and lose focus to the composite base element on mouse leave.',
+			},
 		},
 	},
 };
 
-export const Typeahead: StoryFn< typeof UseCompositeStorePlaceholder > = (
-	storeProps
-) => {
-	const rtl = isRTL();
-	const store = useCompositeStore( { rtl, ...storeProps } );
-
-	return (
-		<Composite store={ store } render={ <Composite.Typeahead /> }>
-			<Composite.Item>Apple</Composite.Item>
-			<Composite.Item>Banana</Composite.Item>
-			<Composite.Item>Peach</Composite.Item>
-		</Composite>
-	);
-};
-Typeahead.parameters = {
-	docs: {
-		description: {
-			story: 'When focus in on the composite widget, hitting printable character keys will move focus to the next composite item that begins with the input characters.',
+export const Typeahead: StoryObj< typeof Composite > = {
+	args: {
+		render: <Composite.Typeahead />,
+		children: (
+			<>
+				<Composite.Item>Apple</Composite.Item>
+				<Composite.Item>Banana</Composite.Item>
+				<Composite.Item>Peach</Composite.Item>
+			</>
+		),
+	},
+	parameters: {
+		docs: {
+			description: {
+				story: 'When focus in on the composite widget, hitting printable character keys will move focus to the next composite item that begins with the input characters.',
+			},
 		},
 	},
 };
+
+// TODO: example accessing the store
+
+// TODO: example across SlotFill forwarding context
```

Comment:
I will add these Storybook examples if we thing that the proposed approach makes sense